### ISSUE TITLE
Indicate level of impact and the group that is impacted

### ISSUE THREAD

**Issue by sophieschoice** [w3c/wai-wcag-em-report-tool#89]
This is a would-be-nice-to-have: with every issue I also document the level of impact (ranging from low to blocking) and the groups that the issue has impact on (users of screen readers, users depending on speech input/commands,...). At the moment the tool does not have this feature.
